fast64_internal/oot/oot_skeleton.py

```diff
@@ -81,13 +81,26 @@ def toC(self):
 				str(self.getNumLimbs()) + " };\n\n" 
 			data.header = "extern SkeletonHeader " + self.name + ";\n"
 
+		index = 0
+		for limb in limbList:
+			name = (self.name + "_" + toAlnum(limb.boneName)).upper()
+			assert limb.index == index
```

Comment:
So, the defines need to follow the order the limbs draw in, which is not necessarily the order they have to be (for the game to work) in `limbList` (aka the `limb.index` value) (`SkeletonHeader#segment`), right?

In this assert, both `index` (order in `limbList`) and `limb.index` (order of exploration by `ootProcessBone`) could in theory not be in the order limbs are drawn in (for example, if `createLimbList` and `ootProcessBone` did breadth-first-search instead of depth-first)

So if this assert is supposed to ensure the indices correspond to the drawing order, it's flawed, I would just remove it, and add a comment to `OOTSkeleton#createLimbList`/`OOTLimb#getList` instead, along the lines of "this needs to be depth-first like the oot skelanime draw code, so the bones are in the order they're drawn in. this is needed for the limb indices defines"

A similar comment may be added to `ootProcessBone` since its traversal should be the same, as "documented" in `OOTLimb#getList`